BUILDIT.md

```diff
@@ -86,6 +86,11 @@ With that in place, let's get building!
 - [9. Update the `LiveView` Template](#9-update-the-liveview-template)
 - [10. Filter Items](#10-filter-items)
 - [11. Tags](#11-tags)
+  - [11.1 Migrations](#111-migrations)
```

Comment:
@LuchoTurtle the docs on this PR are quite incomplete ...
As much as I don't want to "waste" the work done by @SimonLab 
I think it will be _easier_ to use this as "inspiration" 
and build `lists` from scratch so that we can use it in #145

Reply:
Done. ✅  #345 :shipit: